awx_collection/test/awx/test_build_url.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+from unittest import mock
```

Comment:
In pytest, it's customary to use the built-in `monkeypatch` fixture or `mocker` provided by the `pytest-mock` plugin.